apps/admin_app/assets/package.json

```diff
@@ -6,18 +6,26 @@
     "watch": "webpack --mode development --watch-stdin"
   },
   "dependencies": {
+    "@types/jquery": "^3.3.29",
     "@types/prop-types": "^15.5.8",
     "@types/react": "^16.7.18",
     "@types/react-dom": "^16.0.11",
+    "@types/react-router-dom": "^4.3.1",
+    "antd": "^3.13.1",
```

Comment:
please have babel-plugin-import as well so that antd will not bloat up your bundle size.

Reply:
Added it, thanks.